config/set/early-return.php

```diff
@@ -31,6 +31,6 @@
         PreparedValueToEarlyReturnRector::class,
         ReturnBinaryOrToEarlyReturnRector::class,
         ReturnEarlyIfVariableRector::class,
-        // RemoveJustVariableAssignRector::class,
+        RemoveJustVariableAssignRector::class,
```

Comment:
`RemoveJustVariableAssignRector` should be in dead-code instead of early return set

Reply:
Yes, you're right :+1: I forgot the old category

Reply:
moved at https://github.com/rectorphp/rector-src/pull/2621